pyrefinebio/sample.py

```diff
@@ -119,61 +119,81 @@ def experiments(self, value):
 
     @classmethod
     def get(cls, accession_code):
-        """Retrieve a sample based on its accession code.
+        """Retrieve a Sample based on its accession code.
 
         returns: Sample
 
         parameters:
 
-            accession_code (str): The accession code for the sample to be retrieved.
+            accession_code (str): The accession code for the Sample to be retrieved.
         """
         result = get_by_endpoint("samples/" + accession_code)
         return cls(**result)
 
     @classmethod
     def search(cls, **kwargs):
-        """Retrieve a list of samples based on various filters
+        """Retrieve a list of Samples based on various filters
 
-        returns: list of samples.
+        returns: list of Sample
 
         Parameters:
-            ordering (str):                  which field to use when ordering the results
-            title (str):
-            organism (str):
-            source_database (str):
-            source_archive_url (str):
-            has_raw (str):
-            platform_name (str):
-            technology (str):
-            manufacturer (str):
-            sex (str):
-            age (number):
-            specimen_part (str):
-            genotype (str):
-            disease (str):
-            disease_stage (str):
+            ordering (str): which field to use when ordering the results
+
+            title (str): filter based on the Sample's title
+
+            organism (str): filter based on the Organism that the Sample was taken from
+
+            source_database (str): filter based on the database that the Sample was taken from
+
+            source_archive_url (str): filter based on Sample's source url
+
+            has_raw (bool): filter based on if the Sample had raw data available in the source database
+
+            platform_name (str): filter based on the name of the Platform that was used to create the Sample
+
+            technology (str): filter based on the technology that was used to make the Sample
+
+            manufacturer (str): filter based on the manufacturer of the technology that was used to create the Sample
+
+            sex (str): filter based on the sex of the subject that the Sample was taken from
+
+            age (number): filter based on the age of the subject that the Sample was taken from
+
+            specimen_part (str): filter based on the part of the specimen that the Sample was taken from
+
+            genotype (str): filter based on the genotype of the subject that the Sample was taken from
+
+            disease (str): filter based on the disease of the subject that the Sample was taken from
+
+            disease_stage (str): filter based on the stage of the disease of the subject that the Sample
+                                 was taken from
```

Comment:
Similarly...

```suggestion
            age (number): filter based on the age information provided by the submitter

            specimen_part (str): filter based on the part of the specimen reported by the submitter

            genotype (str): filter based on the genotype of the subject that the Sample was taken from

            disease (str): filter based on the disease information provided by the submitter

            disease_stage (str): filter based on the disease stage information provided by the submitter
```

Side note: did we not harmonize `genotype` to `genetic information`? That's what I would expect based on our docs: https://refinebio-docs.readthedocs.io/en/latest/main_text.html#metadata and also is more general than "genotype."

Reply:
@kurtwheeler

Reply:
I made an issue: https://github.com/AlexsLemonade/refinebio/issues/2600